### ISSUE TITLE
Custom selectors and vars syntax producing wrong output

### ISSUE THREAD

**Issue by pvalentim** [postcss/sugarss#5]
Seems that the `:` of the custom selectors and `:root`declaration is breaking something. Notice the `{}` at the end of the color declarations
```stylus
// Input:
@custom-selector :--heading h1, h2
@custom-selector :--any-link :link, :visited

a:--any-link
  color: blue

article :--heading
  color: #000

// Output:
@custom-selector :--heading h1, h2;
@custom-selector :--any-link :link, :visited;

a:--any-link {
  color: blue {}
}

article :--heading {
  color: #000
}
```

css4 custom properties:

```stylus
// Input:
:root
  --red: #F00

body
  color: var(--red)

:root
  --red: #0F0

// Output:
:root {
  --red: #F00
}

body {
  color: var(--red) {}
}

:root {
  --red: #0F0
}
```

The problem here seems to be the double :root, although when using custom selectors and custom properties mixed, the same problem occurs even without the 2 :root declarations.
I've turned all plugins off when testing this btw.

**Comment by ai** [comment]
Oops, I forgot about pseudoclasses :)

Fixed 8916552

**Comment by ai** [comment]
I also clear rule and declaration separation in docs: 47dad04

**Comment by ai** [comment]
Fix was released in 0.1
